test/Parse/ConditionalCompilation/language_version_explicit.swift

```diff
@@ -22,15 +22,15 @@
 #endif
 
 // NOTE: Please modify this condition...
-#if swift(>=4.2)
+#if swift(>=4.2.1)
```

Comment:
Given our discussion with Ted I'm not sure we actually want to bump this number.